### DISCUSSION TITLE
Unable to import kie_predictor

### DISCUSSION THREAD

**Discussion by paulgromy** [mindee/doctr#1270] [Q&A]
I would like to try using KIE predictor as shown in the documentation https://github.com/mindee/doctr#use-the-kie-predictor.
But when I run my code in PyCharm I get an error  "ImportError: cannot import name 'kie_predictor' from 'doctr.models'".

Please let me know what steps I need to take to make the code work. And maybe you will share links to sources where there are examples of the implementation of KIE predictor in docTR

**Comment by felixdittrich92** [comment]
Hi @paulgromy 👋,

this feature is only available on the dev version (main branch) and will be available with the next release. 

If you want to try it you need to clone the repo and install it locally

docTR version: 0.6.1a0

What do you mean with resources of implementations ? How to use it or where the source code is located in docTR ? :)

**Reply by felixdittrich92** [reply]
Links: 
https://mindee.github.io/doctr/latest/using_doctr/custom_models_training.html

https://github.com/mindee/doctr#use-the-kie-predictor

Source: https://github.com/mindee/doctr/tree/main/doctr/models/kie_predictor

**Reply by paulgromy** [reply]
Thanks for the quick response. I thought that perhaps there are code examples with kie_predictor somewhere on the Internet. OK.

Could you please describe to me the specific steps I need to take in order to run the code that is in the documentation?

**Reply by felixdittrich92** [reply]
Mh you mean training your own KIE model or only running the inference ?

For inference:

```
git clone https://github.com/mindee/doctr.git
cd doctr
pip3 install -e .  # [dev] both: tf and pt [tf] or [torch]   eg.: pip3 install -e .[torch]
```
than open a file paste the code snippet, change the document path and run it :)

**Reply by paulgromy** [reply]
Perhaps there is some video on this topic or a detailed tutorial? Because I'm a total newbie and I can't run the code with your hints

**Reply by paulgromy** [reply]
Thanks, I figured out how to install. But in connection with this, I have the following question. Please take a look https://github.com/mindee/doctr/discussions/1275
